connectors/sources/salesforce.py

```diff
@@ -137,6 +155,8 @@ def __init__(self, configuration):
         self._queryable_sobjects = None
         self._queryable_sobject_fields = None
         self._sobjects_cache_by_type = None
+        self._supported_file_types = [x[1:] for x in TIKA_SUPPORTED_FILETYPES]
```

Comment:
Nit: Can probably be inlined

Reply:
Sorry, what do you mean? (I thought inlined means a single line)

Reply:
It's similar to this: https://www.jetbrains.com/help/resharper/Refactorings__Inline_Field.html

If you have a field that's used once in one place, you can instead of assigning `[x[1:] for x in TIKA_SUPPORTED_FILETYPES]` to a field, change the place that uses `self._supported_file_types` to use `TIKA_SUPPORTED_FILETYPES` directly with a filter statement.

Reply:
I didn't realise I was only using it once, that makes sense. Thanks!